spark/src/main/scala/org/apache/spark/sql/delta/DeltaErrors.scala

```diff
@@ -3125,6 +3125,19 @@ trait DeltaErrorsBase
       messageParameters = Array.empty)
   }
 
+  def clusteringWithPartitionPredicatesException(predicates: Seq[String]): Throwable = {
+    new DeltaUnsupportedOperationException(
+      errorClass = "DELTA_CLUSTERING_WITH_PARTITION_PREDICATE",
+      messageParameters = Array(s"${predicates.mkString(" ")}")
+    )
```

Comment:
to be consistent with below
```suggestion
      messageParameters = Array(s"${predicates.mkString(" ")}"))
```